op-node/rollup/derive/l1_block_info.go

```diff
@@ -130,6 +135,9 @@ func L1InfoDeposit(seqNumber uint64, block eth.BlockInfo, sysCfg eth.SystemConfi
 		L1FeeOverhead:  sysCfg.Overhead,
 		L1FeeScalar:    sysCfg.Scalar,
 	}
+	if opservice.ForBSC {
+		infoDat.BaseFee = BSCFakeBaseFee
+	}
```

Comment:
will it in-consist with `func (ba *FetchingAttributesBuilder) prepareAverageGasPrice(ctx context.Context, epoch eth.BlockID) (*big.Int, error)`?